kafka-ui-react-app/src/components/Topics/List/ListItemContainer.ts

```diff
@@ -0,0 +1,35 @@
+import { connect } from 'react-redux';
+import { RootState, ClusterName, TopicName, Action } from 'redux/interfaces';
+import { withRouter, RouteComponentProps } from 'react-router-dom';
+import { ThunkDispatch } from 'redux-thunk';
+import { deleteTopic } from 'redux/actions';
+import ListItem from './ListItem';
+
+interface RouteProps {
+  clusterName: ClusterName;
+}
+
+type OwnProps = RouteComponentProps<RouteProps>;
+
+const mapStateToProps = (
+  state: RootState,
+  {
+    match: {
+      params: { clusterName },
+    },
+  }: OwnProps
+) => ({
+  clusterName,
+});
+
+const mapDispatchToProps = (
+  dispatch: ThunkDispatch<RootState, undefined, Action>
+) => ({
+  deleteTopic: (clusterName: ClusterName, topicName: TopicName) => {
+    dispatch(deleteTopic(clusterName, topicName));
+  },
+});
+
+export default withRouter(
+  connect(mapStateToProps, mapDispatchToProps)(ListItem)
+);
```

Comment:
I'm not sure there is a need to use a container in this case. I would suggest to provide deleteTopic from parent component.